### PR TITLE
User

### PR DESCRIPTION
**Pull request by ShrutiSachpara**

### CODE REVIEW

**Review by krishna2919 — APPROVED** [review]

File: src/app/forgot-password/forgot-password.component.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<body class="bg-forgot">
+  <div class="wrapper">
+    <div
+      class="authentication-forgot d-flex align-items-center justify-content-center"
+    >
+      <div class="card forgot-box">
+        <div class="card-body">
+          <div class="p-4 rounded border">
+            <div class="text-center">
+              <img src="assets/image/forgot-2.png" width="120" alt="" />
```
Comment (SwetaaRampariya) [inline src/app/forgot-password/forgot-password.component.html]: add alt in image tag

File: src/app/forgot-password/forgot-password.component.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <p class="text-muted">
+              Enter your registered email ID to reset the password
+            </p>
+            <div class="my-4">
+              <app-base-input
+                *ngIf="!showOtpInput"
+                label="Email id"
+                [control]="emailControl"
+                [isEmail]="true"
+                [errorMessage]="'Please enter a valid email address'"
```
Comment (SwetaaRampariya) [inline src/app/forgot-password/forgot-password.component.html]: "please enter message " message should be comman and comes from base input.

File: src/app/forgot-password/forgot-password.component.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  sendPasswordReset() {
+    if (!this.showOtpInput && this.emailControl.valid) {
+      this.loading = true;
+      const email = this.emailControl.value!;
+      this.forgotPasswordService
+        .sendPasswordResetEmail(email)
+        .subscribe((response: any) => {
+          this.showMessage(response.message, 'success');
+          if (response.statusCode === 200) {
```
Comment (SwetaaRampariya) [inline src/app/forgot-password/forgot-password.component.ts]: 200 should be come from responseEnums.

File: src/app/loader/loader.component.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  templateUrl: './loader.component.html',
+  styleUrls: ['./loader.component.css'],
+})
+export class LoaderComponent implements OnInit {
+  constructor(private ngxLoaderService: NgxUiLoaderService) {}
+
+  ngOnInit(): void {
+    this.startLoader();
+
+    setTimeout(() => {
```
Comment (SwetaaRampariya) [inline src/app/loader/loader.component.ts:15]: do not laod the loader for static time , it should be be dependent on Api it self.

**Pull request merged**